src/Integration.Vsix/SonarLintTagger/IssueMarker.cs

```diff
@@ -35,32 +35,26 @@ internal class IssueMarker : IFilterableIssue
     {
         public IAnalysisIssue Issue { get; }
         public SnapshotSpan Span { get; }
-        public string WholeLineText { get; set; }
-        public string LineHash { get; set; }
 
         string IFilterableIssue.RuleId => Issue.RuleKey;
 
         string IFilterableIssue.FilePath => Issue.FilePath;
 
-        string IFilterableIssue.LineHash => LineHash;
+        string IFilterableIssue.LineHash => Issue.LineHash;
```

Comment:
IssueMarker doesn't need a dedicated LineHash property anymore: if we created an IssueMarker, it means that we were successful in creating a SnapshotSpan -- which means that the line hash is the same as the issue, and we can just use `Issue.LineHash`